components/common/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+
+import { Pagination, createTheme, ThemeProvider, SxProps, Theme } from '@mui/material'
+import { Container } from '@mui/system'
+
+interface PaginationProps {
+  count?: number
+  onChange: any
+  size?: 'small' | 'medium' | 'large'
+  sx?: SxProps<Theme>
+}
+
+const customTheme = createTheme({
```

Comment:
Don't create a new Material UI theme for a single component. We already have a theme at the root application, make use of it

Reply:
done